test/lit/passes/asyncify_pass-arg=asyncify-addlist@foo_pass-arg=asyncify-removelist@foo.wast

```diff
@@ -0,0 +1,199 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; NOTE: This test was ported using port_passes_tests_to_lit.py and could be cleaned up.
+
+;; RUN: foreach %s %t wasm-opt --asyncify --pass-arg=asyncify-addlist@foo -S --pass-arg=asyncify-removelist@foo -o - | filecheck %s
```

Comment:
This file doesn't use the new flag - what is it testing in this PR?

Reply:
As discussed in https://github.com/emscripten-core/emscripten/issues/13150, this test will check the behavior when a function is matched in both addlist and removelist. I don't know how emitted warnings are ckecked in lit tests.

Reply:
Seems that test/unit/test_asyncify.py is suitable place for this test, adding test_asyncify_addlist_and_removelist to this file.